edge-hub/README.md

```diff
@@ -1,6 +1,17 @@
 # Edge Hub
 This project contains the Edge Hub.
 
+## Startup execution flow
+| Sequence | Description | Code reference |
+|----------|-------------|----------------|
+|1 | Edge hub service starts. | `Program.Main` |
+|2 | Load configuration from appsettings_hub.json and environment variables. | `Program.Main` |
+|3 | Load certificates for Edge hub and install certificate chain if available. | `EdgeHubCertificates.LoadAsync` |
+|4 | Initialize web hosting (ASP.Net Core) and build dependency injection container. | `Hosting.Initialize` > `Startup.ConfigureServices` > `ApplicationDependency.RegisterTo` > module Load method e.g. `CommonModule.Load`, `AmqpModule.Load` |
+|5 | Instantiate each protocol including amqp, mqtt and http if it is enabled. | `Program.GetEdgeHubProtocolHeadAsync`, `AmqpProtocolHead`, `MqttProtocolHead`, `HttpProtocolHead`
```

Comment:
Such documentation which includes method names is very hard to maintain imo. I would suggest we make it more generic, so that it has less chances of quickly becoming stale.

Reply:
Good point.  Updated to reference interface which should not be changed so often.